app/uk/gov/hmrc/eoricommoncomponent/frontend/views/helpers/inputRadioGroup.scala.html

```diff
@@ -51,7 +51,8 @@ <h1 class="heading-large">
         @if(elements.args.get('_hintText).isDefined) {
             <p class="form-hint">@elements.args.get('_hintText)</p>
         }
-        @hintHtml.map(html => html)
+
+        @hintHtml
```

Comment:
Just wondering if we should use a getOrElse on this. To have a string for the html not option of string

But as I remember the behaviour of string will be the same. The none will be treated as no value